Web/keywords/pages/cart_page.robot

```diff
@@ -0,0 +1,27 @@
+*** Keywords ***
+Click Cart
+    Click Element   ${login_locator.btn_cart}
+    Sleep   1
+
+Check No product in cart 
```

Comment:
kw นี้ปอนด์ต้องการให้แค่ check no product in cart หรือถ้าไม่มีให้เพิ่มด้วยนะคะ 

ที่ถูกคือใน level page ควรแค่ check ค่ะ 
ถ้าจะจัดการว่าไม่มีแล้วทำไง ควรเป็น feature